dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
```

Comment:
i'm not sure if it's worth getting into, but you don't have to upload the file first if you're using the CLI, you can just run:
osc process -f filename.json | osc create -f -

whereas if you want to create from an uploaded template, it's:
osc process templatename | osc create -f -